src/Game.java

```diff
@@ -0,0 +1,114 @@
+package src;
+
+
+import java.util.Scanner;
+import java.util.Random;
+
+public class Game {
+    private Player player1;
+    private Player player2;
+
+    public Game(Player player1, Player player2) {
+        this.player1 = player1;
+        this.player2 = player2;
+    }
+
+    public void start() {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Player 1, enter your name:");
+        String player1Name = scanner.nextLine();
+        player1 = new Player(player1Name);
+
+        System.out.println("Player 2, enter your name:");
+        String player2Name = scanner.nextLine();
+        player2 = new Player(player2Name);
+
+        // Let players select their Pokémon
+        selectPokèmon(player1);
+        selectPokèmon(player2);
+
+        // Print selected Pokémon for both players
+        System.out.println(player1.getName() + " selected " + player1.getSelectedPokèmon().getName());
+        System.out.println(player2.getName() + " selected " + player2.getSelectedPokèmon().getName());
+
+        // Implement your battle logic here
+        battle(player1, player2);
+    }
+
+    private void selectPokèmon(Player player) {
```

Comment:
Aangezien dit een actie is die de player doet, zou ik dit verhuizen naar de Player class